src/test/java/com/puppycrawl/tools/checkstyle/internal/utils/XdocGenerator.java

```diff
@@ -68,6 +70,11 @@ public static void generateXdocContent() throws Exception {
             finally {
                 sink.close();
             }
+            final File outputFile = new File(pathToFile.replace(".template", ""));
+            final StandardCopyOption exp = StandardCopyOption.REPLACE_EXISTING;
+            Files.copy(tempFile.toPath(), outputFile.toPath(), exp);
+            tempFile.deleteOnExit();
```

Comment:
this line should be under:
` final File tempFile = File.crea....`

Reply:
Done @romani